### ISSUE TITLE
Feature request: check lockvolumes status

### ISSUE THREAD

**Issue by madkub** [jishi/node-sonos-http-api#474]
Loving the lockVolumes (or as you suggested changing to max volume) but is there a way to see if a zone has the lock applied?

**Comment by jishi** [comment]
No, not at the moment, but it would make sense to add that to the state. However, the locked volume isn't stored in way that makes it easily accessible to present it in the state, so I would need to figure out a better approach for it.

What is your take on the lock vs. max volume? Do you see a point in having both? Or having a min volume?

**Comment by madkub** [comment]
Both the lock and a min/max restriction would be useful to me.

I work in a digital media agency, 8 players around the office. Our office manager has a tendency to set the volume too high, and when people complain it's generally then set too low to maintain ambience. I'm writing a little front-end for this which has a ranger slider min 15%, max 40% - but, having that hard-coded at the api would be much friendlier so we can update the settings.json and set globally (may build an Enterprise iOS app too and don't want to maintain settings twice)

(this front-end also has LDAP login for our AD and groups to control access to functions - Volumes/Playlists/Queue etc)

I'd wanted to have a page with the current volume and a "lock/Unlock" button for each player, but without being able to get the current status, it wouldn't work so well.

**Comment by jgc94131** [comment]
I too would find 'maxvolume' valuable, in fact more valuable than 'lockvolumes'.  I'd prefer it apply per player rather than system wide.
